challenges/Ayush/hard-task-5-design-responsive-dashboard-sidebar/styles.css

```
@@ -0,0 +1,648 @@
+
+    .nav-link:hover {
+        padding-left: 20px;
+    }
+
+    .submenu a {
+        padding: 8px 15px 8px 40px;
+    }
+}
+
+/* Animations */
+@keyframes slideIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)